charts/consul/values.yaml

```diff
@@ -72,7 +72,7 @@ global:
   # image: "hashicorp/consul-enterprise:1.10.0-ent"
   # ```
   # @default: hashicorp/consul:<latest version>
-  image: docker.mirror.hashicorp.services/hashicorppreview/consul:1.17-dev
+  image: hashicorp/consul-enterprise:1.16
```

Comment:
```suggestion
  image: hashicorp/consul-enterprise:1.16-ent
```

Reply:
image: hashicorp/consul-enterprise:1.16-ent 
this dosent work.. it internally appends -ent. so it becomes hashicorp/consul-enterprise:1.16-ent-ent. which is invalid.

Reply:
That's interesting. The acceptance test failures are trying to pull without the `-ent` and failing (logs [here](https://github.com/hashicorp/consul-k8s-workflows/actions/runs/5693468452/job/15432801654))

Reply:
maybe its due to the control plane image being used. mine is built from this branch.